src/commands/weather.js

```diff
@@ -0,0 +1,50 @@
+const { Command } = require("discord-akairo");
+const fetch = require('node-fetch');
+
+class WeatherCommand extends Command {
+    constructor() {
+        super('weather', {
+            aliases: ['weather'],
+            ownerOnly: true,
+            category: 'development',
+            description: {
+                message: "The weather"
+            },
```

Comment:
Try to fit the format of description like the other Commands. The Help command uses information from each description of a Command so we don't have to manually edit the Help command every time. It is required to have a `desc`, `format` and `example` . option is not required.